### PR TITLE
Fix/skill wrapping

### PR DESCRIPTION
**Pull request by litinsky**
1) Added `--no-default-skill` command line param to omit wrapping component to `DefaultStatelessSkill` if component implements `Skill` interface. Is available for `interactbot`, `interactmsbot`, `alexa` modes.

2)  Added `--https` mode to `interactmsbot` mode.

3) Beautified docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,14 +136,15 @@ python -m deeppavlov <mode> <path_to_config> [-d]
...
 * Add section to [*utils/settings/models_info.json*](utils/settings/models_info.json) with your custom Telegram messages
 * In model config file specify `metadata.labels.telegram_utils` parameter with name which refers to the added section of [*utils/settings/models_info.json*](utils/settings/models_info.json)
 
-For the `interactmsbot` mode you should specify **Microsoft app id** in `-i` and **Microsoft app secret** in `-s`. Also before launch you should specify api deployment settings (host, port) in [*utils/settings/server_config.json*](utils/settings/server_config.json) configuration file. Note, that Microsoft Bot Framework requires `https` endpoint with valid certificate from CA.
-Here is [detailed info on the Microsoft Bot Framework integration](http://docs.deeppavlov.ai/en/latest/devguides/ms_bot_integration.html)
-
-You can also store your tokens, app ids, secrets in appropriate sections of [*utils/settings/server_config.json*](utils/settings/server_config.json). Please note, that all command line parameters override corresponding config ones.
+You can also make available DeepPavlov pre-train models in:
```
Comment (yoptar) [inline README.md]:
```suggestion
You can also serve DeepPavlov models for:
```

File: deeppavlov/agents/default_agent/default_agent.py [hunk 2]
```diff
@@ -47,7 +48,7 @@ class DefaultAgent(Agent):
         skills_processor: Initiated agent processor.
         skills_filter: Initiated agent filter.
     """
-    def __init__(self, skills: List[Skill], skills_processor: Optional[Processor]=None,
+    def __init__(self, skills: Union[List[Skill], List[Component]], skills_processor: Optional[Processor]=None,
```
Comment (yoptar) [inline deeppavlov/agents/default_agent/default_agent.py]:
```suggestion
    def __init__(self, skills: List[Component], skills_processor: Optional[Processor]=None,
```
`Skill` is a `Component`, right?

File: deeppavlov/core/agent/agent.py [hunk 3]
```diff
@@ -51,7 +52,7 @@ class Agent(Component, metaclass=ABCMeta):
         dialog_logger: DeepPavlov dialog logging facility.
     """
     def __init__(self, skills: List[Skill]) -> None:
-        self.skills: List[Skill] = skills
+        self.skills: Union[List[Skill], List[Component]] = skills
```
Comment (yoptar) [inline deeppavlov/core/agent/agent.py]: Missed this change in the method definition

File: deeppavlov/agents/default_agent/default_agent.py (hunk 2 above)
Comment (litinsky) [reply deeppavlov/agents/default_agent/default_agent.py]: Fixed

File: deeppavlov/core/agent/agent.py (hunk 3 above)
Comment (litinsky) [reply deeppavlov/core/agent/agent.py]: fixed

**Pull request merged**